packages/cloudformation/templates/givesource-api-donations.yml

```diff
@@ -249,6 +249,43 @@ Resources:
       SourceArn: !Sub arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${RestApi}/*/POST/donations/process
     DependsOn: ProcessDonationsLambdaFunction
 
+  # Post offline donations lambda resources
+
+  PostDonationsOfflineLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        S3Bucket: !Sub |-
+          {{awsLambdaReleaseBucketPrefix}}-${AWS::Region}
+        S3Key: fn/{{version}}/PostDonationsOffline.zip
+      Description: "Post offline donations from admin area."
+      Environment:
+        Variables:
+          AWS_STACK_NAME: !Ref StackName
+      FunctionName: !Sub |-
+        ${StackName}-PostDonationsOffline
+      Handler: "index.handle"
+      MemorySize: 256
+      Role: !Ref LambdaRoleArn
+      Runtime: "nodejs12.x"
```

Comment:
Did you want to update to the latest runtime here or wait and do them all together?

Reply:
I was going to do them all together